Makefile

```diff
@@ -78,6 +79,9 @@ fastsocket_installer:
 nri-device-injector:
 	docker build --pull -t ${REGISTRY}/${DEVICE_INJECTOR_IMAGE}:${TAG} -f nri_device_injector/Dockerfile .
 
+nvidia_persistenced_installer:
```

Comment:
nit: can this be changed to `nvidia-persistenced-installer` to keep inline with style of (most) other make targets?

Reply:
Ah no that makes sense! Thank you for the explanation, lets keep it as it is.